recipes/antlr4-cppruntime/all/CMakeLists.txt

```diff
@@ -1,7 +1,4 @@
 cmake_minimum_required(VERSION 3.0)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
-
-add_subdirectory(source_subfolder/runtime/Cpp)
+add_subdirectory(runtime/Cpp)
```

Comment:
This file is no longer needed, please remove it. Now, CMakeToolchain does all the magic.